src/lib/network/clients/save_network.rb

```diff
@@ -0,0 +1,287 @@
+require "y2storage"
+require "network/install_inf_convertor"
+require "network/network_autoconfiguration"
+require "network/network_autoyast"
+
+module Yast
+  class SaveNetworkClient < Client
+    include Logger
+
+    def main
+      textdomain "network"
+
+      Yast.import "DNS"
+      Yast.import "FileUtils"
+      Yast.import "NetworkStorage"
+      Yast.import "Installation"
+      Yast.import "String"
+      Yast.import "Mode"
+      Yast.import "Arch"
+
+      Yast.include self, "network/routines.rb"
+      Yast.include self, "network/complex.rb"
+
+      # for update system don't copy network from inst_sys (#325738)
+      if !Mode.update
+        save_network
+      else
+        Builtins.y2milestone("update - skip save_network")
+      end
+      # EOF
+
+      nil
+    end
+
+    def adjust_for_network_disks(file)
+      # storage-ng
+      # Check if installation is targeted to a remote destination.
+      devicegraph = Y2Storage::StorageManager.instance.y2storage_staging
+      is_disk_in_network = devicegraph.filesystem_in_network?(
+        Installation.destdir
+      )
+
+      return unless is_disk_in_network
+
+      log.info("Directory \"#{Installation.destdir}\" is on a network based device")
```

Comment:
Whatever message. I just want to leave a trace in the log telling us that we did the check. Logs are the only tool we have to figure out the execution workflow in most bug reports. Even if it's as simple and repetitive as:
```ruby
log.info("Checking whether \"#{Installation.destdir}\" is on a network based device")
return unless is_disk_in_network
log.info("\"#{Installation.destdir}\" is in network")
```
(although sure you can come up with something better) :-)